# He Shen 沈赫

<details>
<summary>Contact Information</summary>

- Tel: +86 18511515898
- Email: <he.shen@outlook.com>
- <details><summary>WeChat: fredsamhaak</summary>
    <img src="../_media/resume-wechat-fredsamhaak.jpeg" alt="wechat-fredsamhaak" width="256" height="256">
    </details>
- ![](../_media/resume-linkedin-logo.png ':size=2%') [Linkedin Profile](https://www.linkedin.com/in/he-shen-8ba5a4aa/)

</details>

## Self Evaluation

Achievement driven. A Team player with strong execution ability.

Collaborating in ways that capitalizes on each partner's strength in complementary ways.

Strong interest in the application of statistics, machine learning and deep learning in the biomedical industry.

## Education

- 2015/09 – 2018/07            [Chinese Academy of Agricultural Sciences](http://www.caas.cn/en/), Master of Science, Vegetable Science

    Supervisor: Prof. Sanwen Huang

- 2011/09 – 2015/07            [China Agricultural University](http://en.cau.edu.cn), Bachelor of Science, Horticulture Science

    Supervisor: Prof. Guojie Liu

## Research in School

- 2015/09 – 2016/08            Bioinformatic analysis of ploidy changes of *Phytophthora infestans*. (Graduate School)

    - Analyzed ploidy changes of *Phytophthora infestans* after treatment using three "adjacent" SNP loci as a set of haplotype markers.

- 2015/04 – 2015/06            Built three different mutation detection pipelines based on WGS technology and compared them with each other. (Undergraduate School)

## Work Experience

### 2022/07 – 2022/12

**Digital Gene AI technology Co. Ltd.**，Bioinformatics Engineer

Responsibilities:

+ Participate in fibrosis project (data analysis).
    - Based on transcriptome data, discover lung, liver, and kidney fibrosis-related genes and potential targets through statistical methods.

+ Participate in the calculation of antibody developability index.
    - Reproduce [the paper of antibody developability index calculation](https://www.pnas.org/doi/10.1073/pnas.1810576116), to judge whether the antibodies obtained in the early stage of drug design have developability problems, and improve the success rate in the later stage based on the calculation of the length of antibody CDR region, hydrophobicity, charge, etc. 

+ Participate in the construction of antigen presentation model.
    - Related articles on reproducing the interaction between MHC and Neoantigen and the likelihood of antigen presentation ([MHC-I](https://pubmed.ncbi.nlm.nih.gov/31844290/), [MHC-II](https://pubmed.ncbi.nlm.nih.gov/31611695/)). Models allow for excellent screening of candidate neoantigens.

+ Participate in research related to HPV detection.
    - Conduct research on the overview of HPV, including its genome, life cycle, etc., as well as detection methods (DNA/RNA, antigen, antibody detection), existing detection products, etc., to pave the way for home detection products.

### 2022/01 – 2022/06 

[**Sanofi**](https://www.sanofi.com), Data Scientist

### 2020/08 – 2021/12 

[**Sanofi**](https://www.sanofi.com), Associate Data Scientist

Responsibilities:

+ RBM (Risk Based Monitoring) work including KRI and DQA analysis so as to find clinical trial sites/regions with abnormal performance on the indicators of concern mainly based on statistical methods.

+ As the main person in charge of the prediction project, explore and promote the application of machine learning and deep learning methods in centralized monitoring.
    - Take 1) AE (Adverse Events) under-reporting and 2) Dropout (subjects withdrawing from clinical trials) as the starting point, build the entire machine learning-based prediction pipeline from scratch which includes, data acquisition, EDA (data exploratory analysis), data cleaning and feature engineering, modeling (hyperparameters tuning, cross-validation, etc.), model evaluation and the final application.
    - In his project, lead and coordinate two colleagues. Through exploration, quick practice (small-step iteration), discussion and sharing within the group, highly enhanced the ability of members.

### 2020/05 – 2020/07 

[**AIAGE**](http://www.aiage.com/index.aspx), Bioinformatics Engineer

Responsibilities:

+ **As the person in charge of algorithm**, built prediction models for health risk assessment from scratch based on 16s and metagenomic data of human intestinal microbes.

    - AIAGE is currently using these models to solve problems such as how likely is a person to get constipation or colorectal cancer.
    - Also, microorganisms (genus/species level) highly correlated to the traits of our interests can be found and their correlation coefficients can be obtained.

### 2018/08 – 2020/04 

[**iCarbonX**](https://www.icarbonx.com/en/), Bioinformatics Engineer/Data Analyst

Responsibilities:

+ As a supporting role, deployed the mutation detection pipeline based on GATK to the production environment.

    - Investigated and built mutation detection pipelines based on GATK and Strelka2 respectively, and compared them with pipeline using BWA (SAMTOOLS).

+ As the main coordinating role, promoted the accomplishment of a dietary recommendation product, which greatly improved the efficiency of dietitians in our company. Patent(No. 201911311565.0) were accepted by SIPO and is now under review.

    - Changed nutrition guidelines into mathematical expressions and wrote codes to implement the core algorithm of the recipes recommendation;
    - As the main coordinating role, organized a 7-person team to design and finalize each module of the product in details and made the product an available one.

+ As one of the three main members, participated in [DREAM Challenges](http://dreamchallenges.org/about-dream/): [Single Cell Signaling in Breast Cancer Challenge](https://www.synapse.org/#!Synapse:syn20366914/wiki/), whose mission was to predict the signaling responses of the breast cancer cell lines to drug treatment at the single-cell level. Won Champions(1/291) in all subchallenges(four in total).
  
    - Focus on subchallenge 3: to predict the signaling responses of the cell lines under the treatment of a brand new drug;
    - Designed data processing and machine learning methods of subchallenges 1, 2 and 4 with other team members. Wrote and reviewed part of the codes.

+ Applied statistical and machine learning methods to multi-omics data mining and found some reported and potential biomarkers.

    - Established correlations between multi-omics and traits using statistical methods such as hypothesis testing between case and control, correlation analysis between biomarkers and traits, and machine learning methods such as classification and regression.

+ Analyzed data of peptide chips based on IST(Immune Signaling Technology), including EDA, correction of batch effects, epitope identification and selection of peptide quantity.

    - The self-developed method was based on dimensionality reduction and removal of batch-related components. It corrected batch effects of the current samples well, and it is expected to achieve good results in incremental analysis;
    - Applied method based on statistics in the literature for epitope identification.
    - The number of peptides is selected based on the greedy search algorithm, hoping to detect as many positive samples as possible with as few peptides as possible.

### 2018/01 – 2018/03 

[**Gese Technology LLP.**](https://www.gesedna.com), Bioinformatics Engineer(Internship)

Responsibilities:

+ Crawl SNP annotation information from databases such as dbSNP.

+ Learn genotype imputation methods. Participate in discussions on scientific design and operation of company's products.

## Skills

- Practical experience of using **Python** for EDA, statistical analysis, machine learning and data visualization (Proficient in using NumPy, Pandas, scikit-learn, SciPy, StatsModels, pingouin, Matplotlib, seaborn, etc.);

- Experience in the use of bioinformatics tools for:
  
    - Mutation detection：BWA, SAMTOOLS, GATK, SOAPsnp, Strelka2
    - Transcriptome analysis：Tophat, Cufflinks, HISAT, StringTie
    - Genome assembly：pbjelly, ARCS
    - Gene annotation：Geneblasta, GeneWise
    - Gene family analysis：OrthoMCL, OrthoFinder, Gblocks, ProtTest, jModeltest, RAxML, PAML, CAFÉ
    - Multiple sequence alignment：MAFFT, MEGA, ClustalX;

- Proficient in using Linux environment and high performance computing clusters.

## Publications

- Li Y, Shen H, Zhou Q, Qian K, van der Lee T, Huang S. [Changing ploidy as a strategy: the Irish potato famine pathogen shifts ploidy in relation to its sexuality.](https://apsjournals.apsnet.org/doi/10.1094/MPMI-08-16-0156-R) Mol Plant Microbe Interact. 2017 Jan;30(1):45-52. doi: 10.1094/MPMI-08-16-0156-R. Epub 2017 Jan 27. PMID: 27957885. (IF: 4.307)
- Liao Q, Du R, Gou J, Guo L, Shen H, Liu H, Nguyen JK, Ming R, Yin T, Huang S, Yan J. The genomic architecture of the sex-determining region and sex-related metabolic variation in Ginkgobiloba. Plant J. 2020 Dec;104(5):1399-1409. doi: 10.1111/tpj.15009. Epub 2020 Oct 27. PMID: 33015884. (IF: 6.417)
- Gabor A, Tognetti M, Driessen A, Tanevski J, Guo B, Cao W, Shen H, Yu T, Chung V; Single Cell Signaling in Breast Cancer DREAM Consortium members, Bodenmiller B, Saez-Rodriguez J. Cell-to-cell and type-to-type heterogeneity of signaling networks: insights from the crowd. Mol Syst Biol. 2021 Oct;17(10):e10402. doi: 10.15252/msb.202110402. PMID: 34661974; PMCID: PMC8522707. (IF: 11.429)


## Languages

+ Foreign language:
    - CET-6 (2016/12, 503)
    - CET-4 (2012/06, 545)
    - IELTS (2011/08, 6.0)

+ First languages:
    - Mandarin & Cantonese

## Honors and Awards

- 2019/12                    [Single Cell Signaling in Breast Cancer Challenge](https://www.synapse.org/#!Synapse:syn20366914/wiki/) organized by [DREAM Challenges](http://dreamchallenges.org/about-dream/); Champions(1/291) in all subchallenges(four in total)

- 2015/12                    September Ninth Movement choral competition of CAAS; act as a chorus conductor; First prize

- 2015/09 – 2017/09        First-class scholarship(twice); Second-class scholarship(Once)

- 2013/11                    The capital university touch Rugby League; Rugby disc champion

- 2013/05                    Sunshine Sports touch Rugby League of CAU; Rugby cup runners up

- 2013/03                    [JA(Junior Achievement)](https://www.juniorachievement.org/web/ja-usa/home); [JA China](http://www.jachina.org/index.html?lang=en_US) Outstanding-volunteer Team

- 2012/06                    The 7th ["The Challenge Cup"](http://tiaozhanbei.net) The capital university student business plan competition; Third prize of CAU

- 2012/06 – 2015/07        Third-class scholarship (Three times)

<p style="text-align: center; font-size: 20px;"><a href=https://fredsamhaak.github.io/CV/#/>[Back Home]</a></p>